src/main/scala/org/zalando/nakadi/client/actor/PartitionReceiverActor.scala

```diff
@@ -34,9 +26,17 @@ object PartitionReceiver{
             automaticReconnect: Boolean,
             objectMapper: ObjectMapper) =
     Props(new PartitionReceiver(endpoint, port, securedConnection, topic, partitionId, parameters, tokenProvider, automaticReconnect, objectMapper) )
+
+  // Actor messages
+  //
+  private object Init
```

Comment:
Very usual for actor messages to use a "token object" like this. Shorter than the sealed case class you had in module scope.